ResearchKit/+Medable/ORKFormItemCell+Medable.m

```diff
@@ -0,0 +1,95 @@
+//
+//  ORKFormItemCell+Medable.m
+//  Axon
+//
+//  Created by me on 5/18/18.
+//  Copyright © 2018 Medable Inc. All rights reserved.
+//
+
+#import "ResearchKit.h"
+#import "ORKFormItemCell.h"
+
+@interface ORKFormItemCell ()
+- (void)ork_setAnswer:(id)answer;
+@end
+
+#pragma mark -
+
+@implementation ORKFormItemTextFieldBasedCell (Medable)
+
+- (void)ork_setAnswer:(id)answer
+{
+    [super ork_setAnswer:answer];
+    
+    @try
+    {
+        // in code below, use KVC, and performSelector:withObject:
+        // in order to bypass compiler/linker restrictions and
+        // avoid adding explicit dependencies to this module (RK)
+        
+        Class DBZxcvbn = NSClassFromString(@"DBZxcvbn");
+        NSString* answerFormatKeyPath = @"answerFormat.";
+
+        NSString* isSecureTextEntryKeyPath =
+        [answerFormatKeyPath stringByAppendingString:@"isSecureTextEntry"];
+        
+        NSString* minPasswordStrengthKeyPath =
+        [answerFormatKeyPath stringByAppendingString:@"minimumPasswordStrength"];
+        
+        int minimumPasswordStrength =
+        [[self.formItem valueForKeyPath:minPasswordStrengthKeyPath] intValue];
+        
+        if (DBZxcvbn && minimumPasswordStrength &&
+            [[self.formItem valueForKeyPath:isSecureTextEntryKeyPath] boolValue])
+        {
+            enum { width = 30 };
+            BOOL answerIsEmpty = [answer isEqual:NSNull.null];
+            UITextField* textField = [self valueForKey:@"textField"];
+            UILabel* indicator = ((UILabel*)textField.leftView ?:
+                                  [[UILabel alloc] initWithFrame:
+                                   CGRectMake(0, 0, width, width)]);
+            
+            textField.leftView = indicator;
+            textField.leftViewMode = UITextFieldViewModeAlways;
+            
+            #pragma clang diagnostic push
+            #pragma clang diagnostic ignored "-Warc-performSelector-leaks"
+            id result = (!answerIsEmpty ?
+                         [[DBZxcvbn new] performSelector:
+                          NSSelectorFromString(@"passwordStrength:")
```

Comment:
- add check if selector exists to avoid the possible exception and the creation of the `DBZxcvbn` instance.

Reply:
- what's the type of answer? DBZ expects a `NSString`, and I couldn't find quickly if that can be `nil`. We could check that in case RK implementation is changed.

Reply:
try block will catch any exception resulting from API change and this selector not being available.

I'll add a check for `answer` being nil, or not `NSString` at top of method.

Reply:
added in https://github.com/Medable/ResearchKit/pull/3/commits/bc5d4f77882b610dfe0d0f256306470b18d86af1